incubator/prisma/values.yaml

```diff
@@ -0,0 +1,97 @@
+# ------------------------------------------------------------------------------
+# Prisma:
+# ------------------------------------------------------------------------------
+
+# The deployment installs one pod by default
+replicas: 1
+
+image:
+  # Prisma image repository
+  repository: prismagraphql/prisma
+  # Prisma image version
+  tag: 1.8
+  # Specify an imagePullPolicy
+  # ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  pullPolicy: IfNotPresent
+
+## Databse configuration for Prisma. In most scenarios, these values
+## don't need to be changed unless you want to use a differnt database
+## than the one in the 'requirementes.yaml' for this chart. If that's the
+## case, please set 'postgresql.enabled' to false at the bottom of this file.
+database:
+  # The current supported connectors are [mysql, postgres]
+  connector: postgres
+  # These are the credentials needed to authenticate with the database.
+  user: prisma
+  password: prisma
+  # If 'postgresql.enabled' is false, you will need to provide the
+  # following values so that Prisma can use them as the database endpoint.
+  # host: ""
+  # port: ""
+
+## Optional secret to authenticate requests done through the Prisma CLI
+auth:
+  enabled: false
+  # secret: ""
+
+## Expose the prisma service to be accessed from outside the cluster (LoadBalancer service).
+## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: ClusterIP
+  port: 4466
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts: []
+    # - prisma.local
+  tls: []
+  #  - secretName: prisma-tls
+  #    hosts:
+  #      - prisma.local
+
+resources: {}
+#  limits:
+#    cpu: 100m
+#    memory: 128Mi
+#  requests:
+#    cpu: 100m
+#    memory: 128Mi
+
+## Node labels for pod assignment
+## ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+## Tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+## Affinity for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+# ------------------------------------------------------------------------------
+# PostgreSQL:
```

Comment:
Is the idea to also support other connectors like mysql?

Reply:
It should be possible to configure other connectors. In order to do so, disable `postgresql` and then set proper configs in the the `database:` section. However, we could also add a MySQL chart as another conditionally dependency, to simplify configurations. What do you think? @emmenko

Reply:
Yeah, although I'm thinking that we might end up with a complex chart if we start supporting all the connectors.

Based also on my [other comment](https://github.com/kubernetes/charts/pull/6483#discussion_r203616442) I would actually suggest to remove the DB chart dependency. I think it's fine to just document what to do. At least as a first step. We could always decide to add it later on if we see a need for that.

Reply:
The existence of the PostgreSQL chart is just for dependency purposes in the same way Zookeeper is to [Kafka](https://github.com/helm/charts/tree/master/incubator/kafka).

I think it's also not a good idea to support all connectors. But it's essential to at least have one set up to ease quickstarts. We don't want users to run in to complex configurations by default, but just leave that as an option for those who want to take a step further.